src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -1091,7 +1091,6 @@
 
 {% block stylesheets %}
   {{ parent() }}
-  <link rel="stylesheet" href="{{ asset('themes/default/css/bundle/module/module.css') }}" />
```

Comment:
If you remove everything in a `{% block %}` but `parent()` you can remove the whole block